todo/appsettings.Development.json

```
@@ -0,0 +1,8 @@
+{
+  "Logging": {
+    "LogLevel": {
+      "Default": "Information",
```

```suggestion
      "Default": "Debug",
```
Develop thì nên để Debug hết nhé